tests/compile-pass/enum.rs

```diff
@@ -1,4 +1,5 @@
-#![feature(lang_items, no_core)]
+// xfail - this makes an unknown error when we try to interpret it in binaryen.
```

Comment:
yeah but isn't this the compile-pass, which is not interpreted ?
does this mean this is now ignored in the compilation tests ?

I'll look into the error